<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1247</width>
    <height>692</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QLabel" name="lblComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>209</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment for new record:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkPG">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Postgres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>A&amp;dd</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sugarCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblInsert">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert Rec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSugar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sugar</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGraph48">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Graph 48 hrs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGraph8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Graph 8 days</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVitals">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>102</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vitals</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblViewRec">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>1221</width>
     <height>361</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnShowBP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>102</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show BP</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtMsgs">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>701</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>102</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
